account_loan_vendor_payment/models/account_payment.py

```diff
@@ -0,0 +1,26 @@
+from odoo import fields, models
+
+
+class AccountPayment(models.Model):
+    _inherit = "account.payment"
+
+    reconciled_bill_ids = fields.Many2many(store=True)
```

Comment:
Why are you making it stored?

You shouldn't do this, because all the other fields of the compute should have the same behaviour and it is obviously a bad decision IMO